overlord/snapstate/snapstate_test.go

```diff
@@ -4932,6 +4929,63 @@ func (s *snapmgrTestSuite) TestEnsureRefreshesWithUpdateStoreError(c *C) {
 	c.Check(autoRefreshAssertionsCalled, Equals, 1)
 }
 
+func (s *snapmgrTestSuite) TestMiscRefresh(c *C) {
```

Comment:
Where do these come from? Looks like new tests? (which is great, just wondering)

Reply:
just to confirm, these are all a chunk of the diff I missed. It passed tests on my end because I still had the other bits lying around. Fixing.